src/RouterSelectorHelper.js

```diff
@@ -0,0 +1,15 @@
+import AsyncStorage from './lib/utils/asyncStorage'
+import { DESTINATION_PATH } from './lib/constants/localStorage'
+import DeepLinking from './lib/utils/deepLinking'
+
+export const savePathToStorage = async (params, log) => {
+  let path = DeepLinking.pathname.slice(1)
+  path = path.length === 0 ? 'AppNavigation/Dashboard/Home' : path
+
+  // TODO: is this: path.indexOf('Marketplace') >= 0 really needed?
+  if ((params && Object.keys(params).length > 0) || path.indexOf('Marketplace') >= 0) {
```

Comment:
@sirpy path.indexOf('Marketplace') >= 0 really needed?
it existed in native but did not exist in web

Reply:
we can reemove